src/pages/MenuPage.js

```diff
@@ -38,96 +95,66 @@ export class MenuPage extends React.Component {
     database.removeListener(this.databaseListenerId);
   }
 
+  /**
+   * Method will try to match stringified customData object for current store
+   * against keys in MODULES, will return values of keys in MODULES
+   * that match: customData[keyInModules]data === 'true'
+   * @param {string}  customDataAsString  Store custom data as string
+   * @return {array}  result              The maxOrdersPerPeriod of the orderType
+   */
+  getUsedModules = customDataAsString => {
+    if (!customDataAsString || customDataAsString.length === 0) return [];
+
+    const customData = JSON.parse(customDataAsString);
+    const result = [];
+    const checkAndAddToResult = ([moduleName, moduleValues]) => {
+      if (!customData[moduleName]) return;
+      if (!customData[moduleName].data) return;
+      if (customData[moduleName].data === 'true') result.push(moduleValues);
+    };
+
+    Object.entries(MODULES).forEach(checkAndAddToResult);
+
+    return result.length > 0 && result;
+  };
+
   exportData = () => {
     const { settings, database } = this.props;
     const syncSiteName = settings.get(SYNC_SITE_NAME);
     database.exportData(syncSiteName);
   };
 
+  renderButton = ({ title, page }) => (
+    <Button
+      style={globalStyles.menuButton}
+      textStyle={globalStyles.menuButtonText}
+      text={title}
+      // eslint-disable-next-line react/destructuring-assignment
```

Comment:
don't like destructuring :(?

Reply:
was trying to avoid making that method have `return` statement, but ended up doing it anyways: destructure